libcodechecker/analyze.py

```diff
@@ -238,6 +238,10 @@ def main(args):
     # Parse the JSON CCDBs and retrieve the compile commands.
     actions = []
 
+    if len(args.logfile) != 1:
```

Comment:
`logfile` has `nargs = '+'` declared, so it is not a string, but an array of strings.
It _could_ be longer. Analyze was supposed to use an arbitrary number of log files (but of course at least one), and from _all_ these build actions, it would generate the output.

(Akin to how `store` and `parse` takes an arbitrary number of plists or plist-folders.)

`check` gives an array of one, because previously, check only operated on a single build command (causing **1** output json) or a single json.